editor/shader_create_dialog.cpp

```diff
@@ -164,37 +164,74 @@ void ShaderCreateDialog::_create_new() {
 			code += vformat("shader_type %s;\n", mode_menu->get_text().to_snake_case());
 
 			if (current_template == 0) { // Default template.
-				code += "\n";
 				switch (current_mode) {
 					case Shader::MODE_SPATIAL:
-						code += "void fragment() {\n";
-						code += "\t// Place fragment code here.\n";
-						code += "}\n";
+						// `vertex()` and `light()` are commented out by default
+						// as these are used less often.
+						// Functions are ordered in the vertex-fragment-light order
+						// to follow established conventions.
+						code += R"(
+//void vertex() {
+	// Called for every vertex the material is visible on.
+//}
+
+void fragment() {
+	// Called for every pixel the material is visible on.
+}
+
+//void light() {
+	// Called for every light affecting the material.
```

Comment:
```suggestion
	// Called for every pixel for every light affecting the material.
```

The same goes for CanvasItem shaders.